lib/modules/datasource/rubygems/get.ts

```diff
@@ -182,4 +208,116 @@ export class InternalRubyGemsDatasource extends Datasource {
 
     return new Marshal(response.body).parsed as T;
   }
+
+  private nexusEndPointRepoFrom(registry: string): {
+    endPoint: string;
+    repository: string;
```

Comment:
```suggestion
    registry?: string;
    repository?: string;
```

Use `registry` instead of `endPoint`. Needs renaming throughout

Reply:
Renamed to `nexusRegistry` since `registry` is being used for a parameter in `getDependencyFallback`.